ccgains/historic_data.py

```diff
@@ -413,5 +413,246 @@ def prepare_request(self, dtime):
 
 
 
+            store.put(key, self.data, format="fixed")
+            return self.data
+
+
+class HistoricDataAPIBinance(HistoricData):
+    last_query_time = pd.Timestamp.now()
```

Comment:
In the end, I probably could/should have moved it back as an instance variable instead of class variable. But when I was writing this I had just finished up a separate project that was pulling down a lot of candlestick data from exchanges across multiple threads, and I needed the last query time to be shared across all instances of the class to ensure none of them broke the rate limits.

But, since we're not doing any parallel operations in ccGains, it ends up having the same effect here, and just leads to some confusion. I'll switch it back to an instance variable